src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/judgeapproval/ApproveDraftOrdersAboutToSubmitHandler.java

```diff
@@ -66,6 +71,7 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
             generateAndStoreCoverLetter(caseDetails, userAuthorisation);
         }
         clearInputFields(draftOrdersWrapper);
+        draftOrderService.clearEmptyOrdersInDraftOrdersReviewCollection(finremCaseData);
```

Comment:
Has this been tested when a draft order is rejected? Does the notification still get sent to the solicitor?